src/utils.ts

```diff
@@ -0,0 +1,103 @@
+import { ethers, Contract, BigNumber } from "ethers";
+import { Finding, FindingSeverity, FindingType, Alert } from "forta-agent";
+import {
+  L2_FUNCTION_SIGNATURE,
+  L2_TOKEN_ADDRESS_MAKER_DAO,
+  previousBalances,
+} from "./constants";
+
+export const getL1Finding = async (
+  daiContract: Contract,
+  blockNumber: number,
+  escrowAddressArbitrum: string,
+  escrowAddressOptimism: string,
+  previousBalances: previousBalances,
+) => {
+  const arbitrumBalance: BigNumber = await daiContract.balanceOf(
+    escrowAddressArbitrum,
+    {
+      blockTag: blockNumber,
+    },
+  );
+  const optimismBalance: BigNumber = await daiContract.balanceOf(
+    escrowAddressOptimism,
+    {
+      blockTag: blockNumber,
+    },
+  );
+
+  if (
+    previousBalances.prevArbitrumBalance != arbitrumBalance &&
+    previousBalances.prevOptimismBalance != optimismBalance
```

Comment:
Two things on this:

1) `!=` doesn't work as expected when comparing BigNumbers, consider using BigNumber comparison methods instead

2) I'd argue that even if one of the balances change, an alert should be raised (so perhaps using `||` instead of `&&`), wdyt?

Reply:
you are right that makes sense. I will adjust accordingly